st2api/st2api/controllers/v1/rules.py

```diff
@@ -52,7 +52,12 @@ class RuleController(resource.ContentPackResourceController):
         'name': 'name',
         'pack': 'pack',
         'action': 'action.ref',
-        'trigger': 'trigger'
+        'trigger': 'trigger',
+        'enabled': 'enabled'
+    }
+
+    filter_transform_functions = {
```

Comment:
We already have at least one place where we do that.

```python

    filter_transform_functions = {
        'system': lambda value: value in ['1', 'true', 'True']
    }
```

We should refactor this function in some API utility function and re-use it in all the splaces so the API is consistent.